### ISSUE TITLE
outdated github cards

### ISSUE THREAD

**Issue by vieux** [moby/mobywebsite#21]
The github cards aren't displayed correctly on the website, see the linuxkit one

<img width="428" alt="screen shot 2017-07-10 at 5 22 50 pm" src="https://user-images.githubusercontent.com/1032519/28025774-926a6dc6-6594-11e7-8513-32b3b027acae.png">

1st, the created by link is blank
2nd, the number of forks / starts is outdated

See the card taken from the repo's test page http://lab.lepture.com/github-cards/:

<img width="497" alt="screen shot 2017-07-10 at 5 23 03 pm" src="https://user-images.githubusercontent.com/1032519/28025758-8a844b0e-6594-11e7-8801-43ca9df6e3b1.png">

**Comment by chanezon** [comment]
I fixed the created link issue https://github.com/moby/mobywebsite/commit/15ebc454ee9a439758d9e5605795000ada4e72ae
For the number of forks and stars, in order to fix https://github.com/moby/mobywebsite/issues/17 and have a faster website, we moved from client side rendering of cards to server side, so they will be updated only when we rebuild the site.
I will need to have a nightly cron job to push a dummy change to the repo, to trigger a rebuild of the site every day to solve it. I'll keep this issue open until I fix that part.
